tests/h/views/api/groups_test.py

```diff
@@ -184,6 +186,62 @@ def pyramid_request(self, pyramid_request, factories):
         return pyramid_request
 
 
+@pytest.mark.usefixtures('group_service',
+                         'user_service',
+                         'request_auth_client',
+                         'validate_auth_client_authority')
+class TestAddMember(object):
+
+    def test_it_adds_user_from_request_params_to_group(self,
```

Comment:
How about we split this into two tests: one that tests that the `member_join` was called and one that tests that it returns an `HTTPNoContent`?